### DISCUSSION TITLE
Problem add adobe digital edition key

### DISCUSSION THREAD

**Discussion by saramartin73** [noDRM/DeDRM_tools#408] [Q&A]
Hi,
with Mac (BigSur version) and DeDRM version v10.0.3 I can't add new Adobe Digital Edition Key: when I click on "plus" sign nothing is happening. The popup asking for the Unique Key Name doesn't open.
Instead with Kindle Serial Number everything works correctly.

Thank you.

**Comment by ElleKayEm** [comment]
Don't try to add the key from DeDRM's customization.  Add an ADE book to calibre and let DeDRM automatically retrieve it.  If you're still having issues, try DeDRM 10.0.9.
